src/Symfony/Component/Yaml/Command/LintCommand.php

```diff
@@ -98,12 +110,25 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->displayCorrectFiles = $output->isVerbose();
         $flags = $input->getOption('parse-tags') ? Yaml::PARSE_CUSTOM_TAGS : 0;
 
+        if ([$this->lintConfig] === $filenames) {
```

Comment:
hmm.. this provides an argument to load a config file, but prevent using any files.

What's about adding a new options `-c | --config` and checking if a `.yamllint.yaml` file exist in the project DIR

Or checking if the $filenames contains a single file and this file is a valid yamlint file

Reply:
> What's about adding a new options `-c | --config`...

I think a new option is a good idea. Maybe with an optional value for a custom config file name `--config=[yamllint.yml]`.

What do you think?

Reply:
I would not provide a default config file, but always require to pass the config filename when this otpion is used.